glean-core/src/metrics/string_list.rs

```diff
@@ -17,13 +18,19 @@ pub struct StringListMetric {
     meta: CommonMetricData,
 }
 
+impl MetricType for StringListMetric {
```

Comment:
He, I explicitly left these untouched for whoever fully implements these types.
I don't mind getting them fixed now though.

Reply:
Yes, I thought about that too, but then I figured that I was copy/pasting outdated code. I think we should either keep things up-to-speed or remove partial implementations. We can have these bits there, the rest is mostly tests and logic that the implementor would need to do anyway.